src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```diff
@@ -70,6 +83,33 @@ export class RequestFeedbackButtonComponent implements OnInit {
         }
     }
 
+    private subscribeToResultUpdates() {
+        if (!this.participation?.id) {
+            return;
+        }
+
+        // Subscribe to result updates for this participation
+        this.athenaResultUpdateListener = this.participationWebsocketService
+            .subscribeForLatestResultOfParticipation(this.participation.id, true)
+            .pipe(
+                skip(1), // Skip initial value
+                filter((result): result is Result => !!result),
+                filter((result) => result.assessmentType === AssessmentType.AUTOMATIC_ATHENA),
+            )
+            .subscribe(this.handleAthenaAssessment.bind(this));
```

Comment:
You also have to unsubscribe again to prevent issues. See the corresponding code rabbit comment